Package.swift

```diff
@@ -17,7 +17,7 @@ let package = Package(
             targets: [
                 networkServiceTargetName,
             ]
-        )
+        ),
```

Comment:
Why is it part of this PR? 🙂

Reply:
Fixed with https://github.com/ios-course/simple-network-service/pull/13/commits/47f42bfe8243951aa4a66fb051fafda289c3a477, but we should add the rules for linters